src/js/samples/components/Create/CreateSample.tsx

```diff
@@ -186,14 +214,17 @@ export function CreateSample({
                 group === "none" ? "" : group,
             );
         } else {
-            onCreate(name, isolate, host, locale, libraryType, subtractionIds, readFiles, labels);
+            onCreate(name, isolate, host, locale, libraryType, subtractionIds, readFiles, labels, group);
```

Comment:
Currently sending group is ignored by the backend unless `forceGroupChoice` is sent. That said I am not against moving towards supporting it in the FE. That said couple of notes:

1. the value of group sent to the backend must be either and int, null, or not sent. 
2. The default value in the form for group should be the current users `primary_group` or `None` iff the user has no `primary_group`

With those two changes you can probably condense the two cases in handleSubmit to a single case